packages/template/package.json

```diff
@@ -19,7 +19,12 @@
   "devDependencies": {
     "@glimmer/component": "^1.1.2",
     "expect-type": "^0.15.0",
-    "sums-up": "^2.1.0"
+    "sums-up": "^2.1.0",
+    "aria-attributes": "^2.0.1",
+    "html-element-attributes": "^3.3.0",
+    "html-event-attributes": "^2.2.0",
```

Comment:
Is there a package for mathml?

Reply:
yes, https://github.com/wooorm/mathml-tag-names
it only has tag names, no attributes exists for them

Reply:
https://github.com/typed-ember/glint/pull/849/files#r2021317938